tools/python/requirements.txt

```diff
@@ -5,7 +5,8 @@ pandas
 tabulate
 requests
 web3
-git+https://github.com/BalancerMaxis/bal_addresses@0.9.1
+git+https://github.com/BalancerMaxis/bal_addresses@main
+git+https://github.com/BalancerMaxis/bal_tools@main
```

Comment:
this addition is problematic for my env manager

i think because setup.py in bal_addresses pins bal_tools, and here you are forcing it to main thus creating a conflict

removing the bal_tools line completely fixes it